src/jsMain/kotlin/team/karakum/showcase/material/inputs/TextField.kt

```diff
@@ -57,6 +59,10 @@ val TextFieldShowcase = FC<Props> {
             label = ReactNode("Start Day")
             variant = outlined
             type = InputType.date
+
+            InputLabelProps = jso {
```

Comment:
Is it according to docs' example?

Reply:
yep https://mui.com/material-ui/react-text-field/#shrink